src/index.ts

```diff
@@ -37,27 +22,20 @@ export function install(container: Element, options?: Partial<Options>) {
   )
   containers.set(container, config)
```

Comment:
I think you've removed all the `containers.get` calls. Can we remove the `WeakMap` entirely?

Reply:
Sadly no(t yet) as there's still this:

https://github.com/github/quote-selection/blob/44e4ced0fb25846fbdfb71aa604d46b136b77a87/src/index.ts#L82